cl/people_db/management/commands/cl_import_judges.py

```diff
@@ -82,6 +82,7 @@ def import_fjc_judges(self, infile=None):
                       'Birth City', 'Birth State',
                       'Death City', 'Death State']
         df = pd.read_csv(infile)
+        df = df.replace(r'^\s+$', np.nan, regex=True) 
```

Comment:
Hideous but necessary, I suppose. So it goes.

Reply:
This one took a little time to figure out.  No idea why there's whitespace in otherwise empty cells in the FJC data but it definitely causes errors when importing.